src/patternfly/components/Menu/menu.scss

```diff
@@ -472,9 +471,10 @@
     padding-inline-end: var(--#{$menu}__list-item--has--menu-action--PaddingInlineEnd);
   }
 
+  &:focus-within,
```

Comment:
I think we need `pf-m-focus` back for typeahead select to manually show visual focus on an item, because actual focus remains on the input.